README.md

```diff
@@ -81,6 +84,26 @@ to get document access.
 - [bit.ly/kueue-controller-design](https://bit.ly/kueue-controller-design)
 presents the detailed design of the controller.
 
+## Roadmap
+
+This is a high-level overview of the main priorities for 2023, in expected order of release:
+
+- Job preemption to reclaim borrowed quota and to accommodate high priority jobs [#83](https://github.com/kubernetes-sigs/kueue/issues/83), this is planned for v0.3
+- Cooperative preemption support for workloads that implement checkpointing [#477](https://github.com/kubernetes-sigs/kueue/issues/477)
+- More advanced allocation strategies for better resource sharing [#312](https://github.com/kubernetes-sigs/kueue/issues/312)
```

Comment:
This is a bit vague, perhaps specifically say: "Tunable flavor assignment to tradeoff cost vs borrowing"

Reply:
Updated, I thought this was merged already. 🙃